### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by kprakobkit**

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -8,9 +8,13 @@ class Home extends Component {
     return (
       dom.div(
         null,
-        dom.h1(null, 'Home'),
-        dom.p(null, Link({ to: '/orders', className: 'orders-link' }, 'Orders')),
-        dom.p(null, Link({ to: '/chef', className: 'chef-link' }, 'Chef'))
+        dom.h1({ className: 'text-center' }, 'pos.'),
+        dom.div(
+          { className: 'row menu' },
+          Link({ to: '/orders', className: 'col-xs-4 col-sm-4 menu-option orders-link' }, 'Orders'),
```
Comment (achiu8) [inline src/components/Home.js]: Do we need both `xs` and `sm` classes?

File: src/components/Nav.js [hunk 2]
```diff
@@ -10,13 +10,19 @@ class Nav extends Component {
         { className: 'clearfix' },
         dom.ul(
           { className: 'nav nav-pills pull-right' },
-          dom.li(null, Link({ to: '/orders', className: 'nav-orders-link' }, dom.span(null, 'Orders'))),
-          dom.li(null, Link({ to: '/chef', className: 'nav-chef-link' }, dom.span(null, 'Chef'))),
-          dom.li(null, Link({ to: '/bartender', className: 'nav-bartender-link' }, dom.span(null, 'Bartender')))
+          this.context.location.pathname.indexOf('/orders') > -1 ?
```
Comment (achiu8) [inline src/components/Nav.js]: Could maybe use ramda `contains`?

File: src/components/Nav.js [hunk 3]
```diff
@@ -10,13 +10,19 @@ class Nav extends Component {
...
+          ) : null,
+          dom.li(null, Link({ to: '/', className: 'nav-home-link' }, dom.span(null, 'Home')))
         )
       )
     );
   }
 }
 
+Nav.contextTypes = {
+  location: PropTypes.object
```
Comment (achiu8) [inline src/components/Nav.js:28]: Where do we use this?

Comment (kprakobkit) [inline src/components/Nav.js:28]: On line 13 - this.context.location. This is gives the child component access to the context passed down from the parent.

Comment (achiu8) [inline src/components/Nav.js:28]: Oh my bad, I read it as `Nav.propTypes`..

File: src/components/Nav.js (hunk 2 above)
Comment (kprakobkit) [inline src/components/Nav.js]: @achiu8 i converted back to using indexOf...for some reason contains doesn't work when deployed to heroku, but works locally...

**Pull request merged**